spring-cloud-gcp-samples/spring-cloud-gcp-logging-sample/src/main/resources/application.properties

```diff
@@ -1 +1,4 @@
 spring.cloud.gcp.logging.enabled=true
+spring.cloud.gcp.logging.headers.xCloudTrace=true
+spring.cloud.gcp.logging.headers.zipkinTrace=true
+spring.cloud.gcp.logging.headers.prioritizeXCloudTrace=true
```

Comment:
Well, my idea was like an `enum`-based single property and indeed exactly only in the auto-configuration module.
And even when we have that property for convenience during external configuration we should leave the room for custom `TraceIdExtractor` bean. Hence `@ConditionalOnMissingBean` before parsing such a property. Although there should be nothing to parse - just `switch` by the `enum` values.